drawing_agent/app/drawing_cache.py

```
@@ -1,96 +1,130 @@
-                    if r.get('metadata', {}).get('drawing_hash') == hash_id]
-        return "\n\n".join(relevant)
\ No newline at end of file
+        query_embedding = self.embed_model.encode(query).tolist()
+
+        # Поиск в FAISS (возвращает List[Tuple[text, score]])
+        search_results = self.vector_db.search(query_embedding, k=top_k)
+
+        relevant_fragments = []
+        target_prefix = f"Drawing_ID:{hash_id}"
+
+        for text, score in search_results:
```

Ruff (0.15.12)

[warning] Loop control variable `score` not used within loop body

Rename unused `score` to `_score`

(B007)